client/js/controllers/loginController.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+featureToggleFrontend.controller('LoginController', ['$scope', function ($scope, $location) {
+    $scope.authProvider = getUrlParameter('authProvider');
+    
+    function getUrlParameter(name){
```

Comment:
i couldn't get the $location.search() to work so instead i've used the direct way of accessing the query string through document.location